<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_base3_cs</class>
 <widget class="QWidget" name="Form_base3_cs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_18">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="0" column="1">
        <widget class="QDateTimeEdit" name="dateTimeEdit_datetimeobs"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Date - time</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Weather</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_meteo"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Subcontractors on site</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_etppres"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_19">
      <attribute name="title">
       <string>Planning</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_16">
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_planning"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>Schedule respect</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>Comments</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextBrowser" name="textBrowser_planningcom">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_20">
      <attribute name="title">
       <string>Presence</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_25">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Headcount</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_18">
          <item row="0" column="0">
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>Contractor headcount</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBox_effectsstraitant"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>Subcontractor headcount</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBox_conduc">
            <property name="text">
             <string>Site supervisor presence</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox_effectetp"/>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="toolButton_effect_etp">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="toolButton_effect_sstraitant">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Machines</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_17">
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBox_eng_cam_deblais">
            <property name="text">
             <string>Dump truck excavation</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBox_eng_cam_aspi">
            <property name="text">
             <string>Vacuum truck</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="checkBox_eng_cam_repand">
            <property name="text">
             <string>Sprayer truck</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="checkBox_eng_cam_malax">
            <property name="text">
             <string>Mixer truck</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="checkBox_eng_compacteur">
            <property name="text">
             <string>Roller</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="checkBox_eng_container">
            <property name="text">
             <string>Container</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_eng_pelle">
            <property name="text">
             <string>Excavator</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox_eng_cam_mat">
            <property name="text">
             <string>Dump truck materials</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_eng_charg">
            <property name="text">
             <string>Loader</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QCheckBox" name="checkBox_eng_finisseur">
            <property name="text">
             <string>Finisher</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBox_eng_trancheuse">
            <property name="text">
             <string>Trencher</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox_eng_compres">
            <property name="text">
             <string>Compressor</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="checkBox_eng_minipelle">
            <property name="text">
             <string>Miniloader</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QCheckBox" name="checkBox_eng_nacelle">
            <property name="text">
             <string>Bucket truck</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QCheckBox" name="checkBox_eng_autre">
            <property name="text">
             <string>Other</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_21">
      <attribute name="title">
       <string>Safety</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_26">
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_20">
          <item row="4" column="2">
           <widget class="QLabel" name="label_42">
            <property name="text">
             <string>Trench shored</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_41">
            <property name="text">
             <string>Trench protected</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QComboBox" name="comboBox_sec_blindage"/>
          </item>
          <item row="3" column="3">
           <widget class="QComboBox" name="comboBox_sec_protectiontranchee"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_40">
            <property name="text">
             <string>Construction site and surroundings cleanness</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_38">
            <property name="text">
             <string>Lighting/signs</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_39">
            <property name="text">
             <string>Waste disposal/sorting</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="comboBox_sec_eclairage"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_37">
            <property name="text">
             <string>Trafic safety</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_sec_personnel"/>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_43">
            <property name="text">
             <string>Markings staked</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_35">
            <property name="text">
             <string>Regulatory documents shown</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_36">
            <property name="text">
             <string>Construction signs</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="comboBox_sec_stockagedechet"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_33">
            <property name="text">
             <string>Staff safety</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_34">
            <property name="text">
             <string>Site protected</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QComboBox" name="comboBox_sec_proprete"/>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="comboBox_sec_circul"/>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_sec_protection"/>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBox_sec_panneauxchant"/>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBox_sec_afficharrete"/>
          </item>
          <item row="0" column="5">
           <widget class="QComboBox" name="comboBox_sec_reunionmarquage"/>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_44">
            <property name="text">
             <string>Markings maintened</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QComboBox" name="comboBox_sec_entretienmarquage"/>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="label_45">
            <property name="text">
             <string>Gaz and water gate valve accessible</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QComboBox" name="comboBox_sec_accesscoupure"/>
          </item>
          <item row="3" column="4">
           <widget class="QLabel" name="label_46">
            <property name="text">
             <string>Work at a height</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QComboBox" name="comboBox_sec_travailhauteur"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Grievance</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_21">
          <item row="0" column="0">
           <widget class="QLabel" name="label_47">
            <property name="text">
             <string>Grievance presence</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_sec_plaintes"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_48">
            <property name="text">
             <string>Grievance object</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_sec_plaintes_com"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
